.github/workflows/build-docs.yml

```
@@ -361,20 +373,23 @@ jobs:
             run: tox c
 
         # Run the doc build
+        #
         # We let tox handle creating virtual env and install dependencies.
         # If no data is restored in previous steps,
         # the data will be downloaded during the build
         # (this only applies for full builds;
         # no data is downloaded for partial builds).
         -   name: Build docs
             id: build-docs
+            if: ${{ matrix.env == 'doc' ||  matrix.env == 'doc_latest' && contains(fromJSON('["pull_request", "workflow_dispatch", "schedule"]'), github.event_name)
```

```suggestion
            if: ${{ matrix.env == 'doc' ||  matrix.env == 'doc_latest' && contains(fromJSON('["workflow_dispatch", "schedule"]'), github.event_name)
```

apply this before merging